### DISCUSSION TITLE
What in the world?

### DISCUSSION THREAD

**Discussion by vasujhawar2001** [cat-milk/Anime-Girls-Holding-Programming-Books#630] [General]
**WHO & WHY?** 
Anyways, now I also feel like contributing, are all these _AI generated_? Not an Anime fan will I get **banned**?

**Comment by karstenx10** [comment]
I do not think any of these are AI generated, and you do not have to be an Anime Fan to enjoy this amazing GitHub Repository. Please feel free to ask me if you have any other questions.

**Comment by cat-milk** [comment]
> are all these _AI generated_?

The shouldn't be we try not to let PRs with obviously Ai generated images through
